pylint/utils/pylint_ast_walker.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's rename the file to `ast_walker` and hopefully in the future we can also drop the `Pylint` prefix from the class.

Reply:
Let's do all of it right now.